src/AssetHandler.js

```diff
@@ -112,6 +112,12 @@ class AssetHandler {
         try {
           return await this.downloadAsset(assetDoc, dstPath)
         } catch (err) {
+          // Ignore missing assets
+          if (err.statusCode === 404) {
```

Comment:
I suggest that we also not fail on 401/403. Similarly, the dataset is somehow odd/a bit broken, but we can still export as much as is accessible to the user.

Reply:
Fair point! Updated.